ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -50,6 +65,9 @@
   private final String queriesFile;
   private final UdfLoader udfLoader;
   private final CountDownLatch shutdownLatch = new CountDownLatch(1);
+  private final ImmutableMap<Class<? extends Statement>, Consumer<Pair<String, Statement>>>
```

Comment:
Replace `Consumer<Pair<blah>>` with `BiConsumer<blah>`

Field should be of type `Map<Blah>` not `ImmutableMap<Blah`.

Is there a reason why you've not made this map `static` and initialized this map inline, as I commented last time?

Reply:
Using `BiConsumer` now and changed the def to `Map`.
Yes, we update `configProperties` in the methods so it cannot be static.